modules/client/src/addressList/internal/client/methods.ts

```diff
@@ -402,45 +402,49 @@ export class ClientAddressListMethods extends ClientCore
         ),
       );
     } catch {
-      throw new GraphQLError("AddressList proposals");
+      throw new GraphQLError("AddresslistVoting proposals");
     }
   }
 
   /**
    * Returns the settings of a plugin given the address of the plugin instance
    *
    * @param {string} pluginAddress
-   * @return {*}  {(Promise<IPluginSettings | null>)}
+   * @return {*}  {(Promise<VotingSettings | null>)}
    * @memberof ClientAddressListMethods
    */
-  public async getSettings(
+  public async getVotingSettings(
     pluginAddress: string,
-  ): Promise<IPluginSettings | null> {
+  ): Promise<VotingSettings | null> {
     if (!isAddress(pluginAddress)) {
       throw new InvalidAddressError();
     }
     try {
       await this.graphql.ensureOnline();
       const client = this.graphql.getClient();
-      const { addresslistPlugin } = await client.request(
-        QueryAddressListPluginSettings,
+      const { addresslistVotingPlugin }: {
+        addresslistVotingPlugin: SubgraphVotingSettings;
+      } = await client.request(
+        QueryAddressListVotingPluginSettings,
         {
           address: pluginAddress,
         },
       );
-      if (!addresslistPlugin) {
+      if (!addresslistVotingPlugin) {
         return null;
       }
       return {
-        minDuration: parseInt(addresslistPlugin.minDuration),
-        minSupport: decodeRatio(
-          parseFloat(addresslistPlugin.totalSupportThresholdPct),
-          2,
+        minDuration: parseInt(addresslistVotingPlugin.minDuration),
+        supportThreshold: parseFloat(
+          formatEther(addresslistVotingPlugin.supportThreshold),
+        ),
+        minParticipation: parseFloat(
+          formatEther(addresslistVotingPlugin.minParticipation),
```

Comment:
Why not our own way to decode ratios? these values are not an ERC20/eth money value

There will be the follow-up task for that anyway

Reply:
Yes there is a follow up task on this. Our encodeRatio and decodeRatio only support 15 digit precission whit at the time was good enough for us. at this moment we need way moe thant that since on the contracts for example 0.5 will be stored as 5*10^17. Both `parseEther` and `formatEther`give use the functionality of going from  0.5 yto 5*10^17 and viceversa so I think there is no need of reinventing the wheel and we can drop our `encodeRatio` and `decodeRatio`